examples/temporal/main.js

```diff
@@ -0,0 +1,571 @@
+
+            var ground = osg.createTexturedBoxGeometry( 0.0, 0.0, 0.0, size, size, size );
+            ground.setName( 'groundBox' );
+
+            var groundPlace = new osg.MatrixTransform();
+            groundPlace.addChild( ground );
+            var m = groundPlace.getMatrix();
+            osg.Matrix.makeRotate( 0.0, 0.0, 0.0, 1.0, m );
+            osg.Matrix.setTrans( m, 0.0, 0.0, 0.0 );
+            group.addChild( groundPlace );
+
+            ground.getOrCreateStateSet().setTextureAttributeAndModes( 0, osg.Texture.createFromURL( this._mediaPath + 'textures/seamless/bricks1.jpg' ) );
+
+
+            var gridDefinition = 15;
+            // THE GRID http://www.youtube.com/watch?v=S-d7iXuJlvE&t=0m9s
+            var theGrid = osg.createGridGeometry( -1.0, -1.0, 0.0, 2.0, 0.0, 0.0, 0.0, 2.0, 0.0, gridDefinition, gridDefinition );
+
+            var material = new osg.Material();
+            theGrid.getOrCreateStateSet().setAttributeAndModes( material );
+            material.setEmission( [ 1.0, 1.0, 1.0, 1.0 ] );
+            material.setDiffuse( [ 1.0, 1.0, 1.0, 1.0 ] );
+            material.setSpecular( [ 1.0, 1.0, 1.0, 1.0 ] );
+            material.setAmbient( [ 1.0, 1.0, 1.0, 1.0 ] );
+
+            group.addChild( theGrid );
+
+            return group;
+        },
+
+        ///// UTILS
+
+        // show the renderTexture as ui quad on left bottom screen
+        // in fact show all texture inside this._rtt
+        showHideFrameBuffers: function ( optionalArgs ) {
+
+            // debug Scene
+            if ( !this._rttDebugNode ) {
+                this._rttDebugNode = new osg.Node();
+                this._rttDebugNode.setName( '_rttDebugNode' );
+                this._root.addchild( this._rttDebugNode );
+            } else if ( this._rttDebugNode.getChildren().length !== 0 ) {
+                this._rttDebugNode.removeChildren();
+                return;
+            }
+
+            var _ComposerdebugNode = new osg.Node();
+            _ComposerdebugNode.setName( 'debugComposerNode' );
+            _ComposerdebugNode.setCullingActive( false );
+            var _ComposerdebugCamera = new osg.Camera();
+            _ComposerdebugCamera.setName( '_ComposerdebugCamera' );
+            this.rttDebugNode.addChild( _ComposerdebugCamera );
+
+            var optionsDebug = {
+                x: 0,
+                y: 100,
+                w: 100,
+                h: 80,
+                horizontal: true,
+                screenW: 1024,
+                screenH: 768,
+                fullscreen: false
+            };
+            if ( optionalArgs )
+                osg.extend( optionsDebug, optionalArgs );
+
+            var matrixDest = _ComposerdebugCamera.getProjectionMatrix();
+            osg.Matrix.makeOrtho( 0, optionsDebug.screenW, 0, optionsDebug.screenH, -5, 5, matrixDest );
+            _ComposerdebugCamera.setProjectionMatrix( matrixDest ); //not really needed until we do matrix caches
```

Comment:
oups whole RTT debug thing is a code copy/paste from 2 yrs ago.
( Refrained to copy it inside the Example, but would have liked to )